### ISSUE TITLE
No new videos detected on playlists

### ISSUE THREAD

**Issue by w0d4** [w0d4/yt-backup#45]
**Describe the bug**
No new videos where detected on playlists, since the youtube API changed a bit and we cannot get etags for multiple playlists in one request now.

**Additional context**
API usage must be changed. So each playlists etag must be checked on it's own.
This will use much more API quota.

**Comment by swissalps9** [comment]
please explain what you are experiencing? works fine for me, I can still get etags for multiple playlists.

**Comment by w0d4** [comment]
OK, now the things got strange.
I wondered that nobody opened a issue before me.
I saw, that since the 12th December, no more videos where loaded.
I debugged this a bit and found the issue in this function: https://github.com/w0d4/yt-backup/blob/0a85c7900bc993bcbe3658c439becb848a6aa8af/yt-backup.py#L717

In line 733 we see the API youtube.playlists().list is called with a comma seperated list of playlists I want to check.
https://github.com/w0d4/yt-backup/blob/0a85c7900bc993bcbe3658c439becb848a6aa8af/yt-backup.py#L733

At the time I had written this function, I got a list of playlists with their etags as answer. Yesterday I get nothing of use back from the API.

When I checked the following two playlists "UUAKB9wGGRccBDrvbbbmgetw,UUvFnURaRcZ_fdpmrPhICEgg" I got the following result:

```
{
  "kind": "youtube#playlistListResponse",
  "etag": "PrVAaZWyqsAXeT7fHGdWmULDRGI",
  "pageInfo": {
    "totalResults": 0,
    "resultsPerPage": 5
}
```
As I checked it right now, everything seems back to normal. This is very strange for me.
I will leave this open for a few days and monitor if the problem is appearing again.
If not, I will close the issue.
